2.0.0-Data-Model.md

```diff
@@ -12,7 +12,9 @@ We are starting with a read-only API due to potentially different views and appr
 
 Multiple formats for descriptors such as [CWL](https://github.com/common-workflow-language/common-workflow-language) and [WDL](https://github.com/broadinstitute/wdl) are permitted. 
 
-Tools should have a globally unique identifier which identifies the authoritative source for a tool (ex: 123456323@agora.broadinstitute.org )
+Tools IDs can be prefixed and exchanged in a CURIE-like format instead of URLs to identify both the TRS implementation that they originated from and to provide a stable ID that is forward compatible  (ex: `dockstore:123456323` could resolve to https://dockstore.org/api/ga4gh/v1/tools/123456323, https://dockstore.org/api/ga4gh/trs/v2/tools/123456323, https://dockstore.org/api/ga4gh/trs/v3/tools/123456323, etc. ). To define the prefixes and the TRS implementations that they correspond to, we recommend that TRS implementations add themselves to https://github.com/ga4gh/tool-registry-service-schemas/blob/develop/registry.json to provide for the possibility of creating a global resolver.
+
+In the case of private TRS implementations not known to the public, we only recommend consulting the same registry.json to determine a unique prefix to avoid confusion in the case of collisions. 
```

Comment:
The `only` in `only recommend` sounds odd to me. I would either just say `...we recommend...`, or just even drop the recommend phrase entirely and say `...to the public, consult the same registry.json...`.

All subjective and I don't feel strongly if you disagree.